airflow/operators/python_operator.py

```diff
@@ -86,6 +87,38 @@ def __init__(
         if templates_exts:
             self.template_ext = templates_exts
 
+    def __deepcopy__(self, memo):
+        """
+        Hack sorting double chained task lists by task_id to avoid hitting
```

Comment:
Copy pasted?

Reply:
will remove this. the function is copied from BaseOperator, but modify based on the need.